marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/builtins/steps/mapping/entity-services/runEmptyMapping.sjs

```diff
@@ -0,0 +1,35 @@
+const mapping = require("/data-hub/5/builtins/steps/mapping/entity-services/main.sjs");
+const test = require("/test/test-helper.xqy");
+
+let assertions = [];
+function runStructuredPropertyMapping () {
+  let doc = fn.head(xdmp.unquote(`{"id":1,"transactionDate":"12/7/2019","customer":{"firstName":"Oralia","lastName":"Dinesen","gender":"F"},"items":[{"name":"Voltsillam","price":2.0,"quantity":7},{"name":"Latlux","price":7.17,"quantity":10},{"name":"Biodex","price":5.01,"quantity":2},{"name":"Fixflex","price":8.77,"quantity":6},{"name":"Keylex","price":5.57,"quantity":3}]}`));
```

Comment:
Let's simplify the input doc to the bare minimum here - no transactionDate needed, and only need one item.

Reply:
Removed transactionDate, added 2 items so as to ensure all instances of it get removes from '$results' obtained by calling xdmp:xslt-invoke in line 64.
```
<?xml version="1.0" encoding="UTF-8"?>
<OrderType xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:map="http://marklogic.com/xdmp/map" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <id xsi:type="xs:string">1</id>
  <customer>
    <CustomerType/>
  </customer>
  <items datatype="array">
    <ItemType/>
  </items>
  <items datatype="array">
    <ItemType/>
  </items>
</OrderType>
```